### ISSUE TITLE
Bootstrap Hamburger Menu not working

### ISSUE THREAD

**Issue by Davedoo** [Office-of-Digital-Innovation/California-State-Template-Web-Forms#1]
I just download the project here and run it in Visual Studio. What came to my attention is that the Bootstrap Hamburger Menu not working when webform is in mobile view.

Any idea why?

Thanks,
Dave

**Comment by kkoryaka** [comment]
Good catch. We accidentally used beta core Js file v5.5.3 instead of v5.0.6. We replaced it with v5.0.6, Everything is working properly now. Thanks for bringing it to our attention.

**Comment by Davedoo** [comment]
Thanks for fast reply! Your solution works only in Microsoft Edge. It is still not working in IE and Chrome.
I appreciate if you can solve this for all browsers.

**Comment by kkoryaka** [comment]
I just downloaded the latest release (from here https://github.com/Office-of-Digital-Innovation/California-State-Template-Web-Forms/releases) and  tested it in all common browsers. Seems like it's working fine.

**Comment by Davedoo** [comment]
Yes it is now working fine indeed. Perhaps was a caching problem! thanks!

**Comment by Davedoo** [comment]
One more question, what is the best way to add "Ask Bar"  just like the html template under "Featured Search Box"? I am following readme.txt from html sample but have some difficulties to get it quite neatly inside Site.Master. thanks again!

**Comment by kkoryaka** [comment]
You will find instructions and examples on this page: http://template.webstandards.ca.gov/sample/featured-search.html
